tdrs-frontend/src/actions/createXLSReport.test.js

```diff
@@ -0,0 +1,21 @@
+import { getParseErrors } from './createXLSReport'
+
+it('should create an action to create an XLS report', () => {
+  const data_json = {
+    data: {},
+    xls_report: 'aGVsbG8=',
+  }
+  // atob is not available in jest
+  const expectedReturn = Error(
+    'TypeError: URL.createObjectURL is not a function'
```

Comment:
i've only looked at this for a bit, but the solutions seem to be either
1. install the `base-64` package and use those methods instead  - https://stackoverflow.com/a/51525605
2. expose `atob` in the jest vm - https://stackoverflow.com/a/70962105
3. mock the function call and response, like we've already talked about

i think it'd be worth experimenting with to make sure we have sufficient test coverage. Option 2 actually seems like a relatively low complexity solution and we don't have to mock anything.